css/style.css

```diff
@@ -0,0 +1,100 @@
+
+
+body {
+	font-family: Arial;
+	background: black;
+  -webkit-font-smoothing: antialiased;
+}
+
+hgroup { 
```

Comment:
It looks like the `hgroup` element has been [removed from the HTML5 spec](http://html5doctor.com/the-hgroup-element/), so even though it might still work, there's no guarantee that it will work properly (or at all) on all browsers.